openshift/system/config/settings.yml

```diff
@@ -1,12 +1,10 @@
-base: &default
+production:
   superdomain: <%= superdomain = ENV.fetch('THREESCALE_SUPERDOMAIN', 'example.com') %>
   apicast_internal_host_regexp: '\Asystem-(master|provider|developer)\Z'
   secure_cookie: true
   force_ssl: true
   apicast_oauth: true
   apicast_custom_url: true
-  apicast_configuration_driven: true
```

Comment:
these two àpicast_configuration_driven` and `active_docs_url` are not used. 

Usage for the latter was removed here: https://github.com/3scale/system/commit/90ebb558ef168550289c17dc8b3a4123cd3f2987#diff-67dc91a359da89ceb6c89092f4797e95a1ba6c16a5b915de4a560ae68e61a56f